bob/src/api/http.rs

```diff
@@ -87,9 +87,42 @@ pub(crate) struct DiskState {
     is_active: bool,
 }
 
+#[derive(Debug, Serialize)]
+pub(crate) struct Version {
+    version: String,
+    build_time: String,
+}
+
+#[derive(Debug, Serialize)]
+pub(crate) struct VersionInfo {
+    bob_version: Version,
+    pearl_version: Version,
+}
+
+pub fn get_bob_version() -> String {
```

Comment:
I suggest to move this helpers to separate mod, because you use them outside of `bob` crate

Reply:
Done